PanCardView/CardsView.cs

```diff
@@ -697,22 +697,22 @@ protected override void OnSizeAllocated(double width, double height)
         {
             base.OnSizeAllocated(width, height);
 
-            lock (_sizeChangedLocker)
```

Comment:
no need to drop this lock

Reply:
Okay i fix it but lock in ui method is not performance sometime if the method calls a lot of times.